StevenJohnson12

```diff
@@ -0,0 +1,36 @@
+---
+name: "@StevenJohnson12 (Steven Johnson)"
+project: "Custom auto alert door security system"
+---
+
+# Door security system
+
+## Summary
+
+My home has been broken into 2 times in the last year when I was not home. I don't currently have a proper security system installed for my doors & windows, because my parents are unable to afford one. So for my project I plan on creating an automatic wireless alert system for my front door using a Flipper Zero device to receive input from a door sensor & then use the radio frequency transmitter on the flipper to send out a signal to my Arduino Uno with a radio receiver antenna connected to my wifi network, running an automatic notification system to automatically send me a text message to my phone using the twilio API. I will then be notified by text every time the door is opened when i'm not home and hopefully this will prevent and future break-ins. 
+
+## Plan
+
+What steps are you going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+1. Buy all required items listed in the budget. I already own most of the items needed for this project (connectors, breadboards, many other arduino components. 
```

Comment:
please include all the items in your budget (with a price of 0$), as we want to keep an accurate bill of materials